src/types.ts

```diff
@@ -71,13 +72,15 @@ export interface Device {
 }
 
 export interface HealthCertDocument extends v2.OpenAttestationDocument {
-  name: string;
+  version?: string;
+  type?: string;
+  name?: string;
   validFrom: string;
   fhirVersion: string;
   logo: string;
   fhirBundle: {
     entry: [Patient | Observation | Specimen | Organisation | Device];
```

Comment:
When v1 is phased out, we should remove ~~`entry: [Patient | Observation | Specimen | Organisation | Device];`~~ and just use R4.IBundle.

Should also remove all the manually defined interfaces (line 7 to 72) that are defined in this file because they are not FHIR compliant.

Reply:
ok.. let me add deprecated comment info for these interfaces